vault/resource_okta_auth_backend.go

```diff
@@ -76,17 +80,23 @@ func oktaAuthBackendResource() *schema.Resource {
 			},
 
 			"ttl": {
-				Type:        schema.TypeString,
-				Required:    false,
-				Optional:    true,
-				Description: "Duration after which authentication will be expired",
+				Type:         schema.TypeString,
+				Required:     false,
+				Optional:     true,
+				Default:      "0",
```

Comment:
If `ttl` and `max_ttl` are optional, should they even have a default value here on the terraform side?

Reply:
Good question, without the defaults Terraform will detect a difference between the configured auth backend after an `apply`.  I will investigate further to see if there is a better approach that satisfies this use case.

Reply:
Fixed in [185d8311](https://github.com/hashicorp/terraform-provider-vault/pull/1123/commits/185d8311c0c99e9fcff665c128f310271bdddaae)